packages/loaders/src/Skeleton.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+import { rgba } from 'polished';
+import { retrieveTheme, isRtl } from '@zendeskgarden/react-theming';
+import {
+  zdColorGrey800,
+  zdColorGrey100,
+  zdColorKale800,
+  zdColorWhite,
+  zdSpacingXxs
+} from '@zendeskgarden/css-variables';
+
+const COMPONENT_ID = 'loaders.skeleton';
+
+const fadeInAnimation = keyframes`
+  0%, 60% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+`;
+
+const skeletonAnimation = keyframes`
+  100% {
+    left: 100%;
+  }
+`;
+
+const skeletonRtlAnimation = keyframes`
+  100% {
+    right: 100%;
+  }
+`;
+
+/* eslint-disable */
+const StyledSkeleton = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  display: inline-block;
+  position: relative;
+  animation: ${fadeInAnimation} 750ms linear; /* stylelint-disable-line */
```

Comment:
`stylelint-disable` shouldn't be needed here (I was able to remove without issue).

Reply:
It seems `stylelint` updated their [globby](https://github.com/sindresorhus/globby) dependency. This requires all globs to be wrapped in `"`.

Updating our `lint:css` script triggered the warnings. I removed 2 rules that continually get flagged due to prettier and styled-components usage. This should be now be accurate.